docs/document-level-security/SERVICENOW.md

```diff
@@ -0,0 +1,35 @@
+### Setting up the ServiceNow connector
+
+See the [Developer guide](../../docs/DEVELOPING.md) for setting up connectors.
+
+## Document level security
+
+Document level security (DLS) enables you to restrict access to documents based on a user's permissions. This feature is available by default for the ServiceNow connector.
+
+Refer to [document level security](https://www.elastic.co/guide/en/enterprise-search/master/dls.html) for more information.
+
+ServiceNow connector support ACL roles for document level security. For default services, connectors use the below roles to find users who have access to documents.
+
+| Service        | Roles                                                             |
+|----------------|-------------------------------------------------------------------|
+| User           | admin                                                             |
+| Incident       | admin, sn_incident_read, ml_report_user, ml_admin, itil           |
+| Requested Item | admin, sn_request_read, asset, atf_test_designer, atf_test_admin  |
+| Knowledge      | admin, knowledge, knowledge_manager, knowledge_admin              |
+| Change Request | admin, sn_change_read, itil                                       |
+
+For services other than default, connector iterate over access controls with read operations and find the respective roles of the services.
```

Comment:
```suggestion
For services other than the defaults, the connector iterates over access controls with read operations and finds the respective roles for those services.
```

Do we support non-default services though? I thought we only supported these. Or is this an Advanced Sync Rules edge case?

Reply:
Yes, we are supporting non-default services as well.